packages/preview/tuhi-exam-vuw/0.1.0/typst.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tuhi-exam-vuw"
```

Comment:
The name of the package should not be the most obvious/canonical one for what the package does. Can you please rename it to something else?

Reply:
I was hoping to have at least some consistency with [tuhi-course-poster-vuw](https://typst.app/universe/package/tuhi-course-poster-vuw). Note that `tuhi` is a Maori word that is never used in English, so I really can't imagine anyone would come up with this same package name, or mistake it for a canonical name.